ast/edit/testcase_test.go

```diff
@@ -74,3 +81,134 @@ myVar = 4`
 		t.Errorf("unexpected package count: want: 0, got: %d", len(transformed))
 	}
 }
+
+func TestTestcaseTransformImport(t *testing.T) {
+	fs := &inmemFS{
+		files: map[string]string{
+			"a/a_test.flux": `package a_test
+import "testing/assert"
+want = 4
+testcase a {
+	assert.equal(want: want, got: 2 + 2)
+}
+`,
+			"b/b_test.flux": `package b_test
+import "testing/assert"
+testcase b {
+	import "../a/a_test"
+	a_test.a()
+	assert.equal(want: 6, got: 3 + 3)
+}
+`,
+		},
+	}
+
+	ctx := filesystem.Inject(context.Background(), fs)
+	data, err := filesystem.ReadFile(ctx, "b/b_test.flux")
+	if err != nil {
+		t.Fatal(err)
+	}
+	pkg := parser.ParseSource(string(data))
+	if ast.Check(pkg) > 0 {
+		t.Fatalf("unexpected error parsing b/b_test.flux: %s", ast.GetError(pkg))
+	}
+	pkg.Files[0].Name = "b/b_test.flux"
+
+	names, pkgs, err := edit.TestcaseTransform(ctx, pkg)
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+
+	if want, got := []string{"b"}, names; !cmp.Equal(want, got) {
+		t.Fatalf("unexpected testcase names -want/+got:\n%s", cmp.Diff(want, got))
+	}
+
+	files := make([]string, len(pkgs[0].Files))
+	for i, file := range pkgs[0].Files {
+		files[i] = ast.Format(file)
+	}
+
+	want := []string{
+		`package main
+
+
+import "testing/assert"
+
+a_test = () => {
+	want = 4
+	a = () => {
+		assert.equal(want: want, got: 2 + 2)
+
+		return {}
+	}
+
+	return {want, a}
+}()`,
+		`package main
+
+
+import "testing/assert"
+
+a_test.a()
+assert.equal(want: 6, got: 3 + 3)`,
+	}
+	if got := files; !cmp.Equal(want, got) {
+		t.Fatalf("unexpected file contents -want/+got:\n%s", cmp.Diff(want, got))
+	}
+}
+
+type inmemFS struct {
```

Comment:
Could we expand this out to `inMemoryFilesystem`? I don't thing it buys us anything to abbreviate things here.